### PR TITLE
Add leagues

### PR DESCRIPTION
**Pull request by 15lnestor**
## Description of Change
Adds the League model to the application. Also includes basic validation specs. 

## Alternate Designs
N/A

## Benefits
N/A

## Possible Drawbacks
N/A

### CODE REVIEW

File: spec/models/league_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe League, type: :model do
```
Comment (scottmascio2115) [inline spec/models/league_spec.rb]: Rspec! Awesome!

File: spec/models/league_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  before do
+    @league = create(:league)
+  end
+
+  it "should be valid" do
```
Comment (scottmascio2115) [inline spec/models/league_spec.rb]: Use single quotes in ruby unless you are doing string interpolation.

File: spec/models/league_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    expect(@league).to be_valid
+  end
+
+  describe "name" do
```
Comment (scottmascio2115) [inline spec/models/league_spec.rb]: I think you could do this with less code look [here](https://makandracards.com/makandra/38645-testing-activerecord-validations-with-rspec)

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -28,6 +28,8 @@
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
 
+
```
Comment (scottmascio2115) [inline spec/spec_helper.rb]: No need for this extra white space

File: Gemfile [hunk 5]
```diff
@@ -21,7 +21,7 @@ gem 'jbuilder', '~> 2.5' # Build JSON APIs with ease. Read more: https://github.
 # gem 'capistrano-rails', group: :development # Use Capistrano for deployment
 
 gem 'figaro'
-gem 'haml' # Do we need this?
+# gem 'haml' # Do we need this?
```
Comment (scottmascio2115) [inline Gemfile]: This gem https://github.com/indirect/haml-rails

**Comment by 15lnestor** [comment]
I wasn't sure if your first comment was the "go ahead" or not, so that's why I merged. I'll add these changes to the next PR.

**Comment by scottmascio2115** [comment]
Oh np. I started the review then got distracted and came back to it. It musta got in a weird state. Looking good so far

**Pull request merged**